src/functions/usernameToColor/usernameToColor.spec.js

```diff
@@ -0,0 +1,43 @@
+import usernameToColor from './index'
```

Comment:
Quick note though, wouldn't it better to put them in /tests ?

Reply:
@juliushaertl

Reply:
Definitely, pushed a commit for that.